Source/SwiftLintFramework/Rules/NimbleOperatorRule.swift

```diff
@@ -39,9 +39,20 @@ public struct NimbleOperatorRule: ConfigurationProviderRule, OptInRule {
             "↓expect(10).to(beLessThanOrEqualTo(10))\n",
             "↓expect(x).to(beIdenticalTo(x))\n",
             "expect(10) > 2\n ↓expect(10).to(beGreaterThan(2))\n"
+        ],
+        corrections: [
+            "↓expect(seagull.squawk).toNot(equal(\"Hi\"))\n": "expect(seagull.squawk) != \"Hi\"\n",
+            "↓expect(\"Hi!\").to(equal(\"Hi!\"))\n": "expect(\"Hi!\") == \"Hi!\"\n",
+            "↓expect(12).toNot(equal(10))\n": "expect(12) != 10\n",
+            "↓expect(value1).to(equal(value2))\n": "expect(value1) == value2\n",
+            "↓expect(value1).to(equal(10))\n": "expect(value1) == 10\n",
+            "↓expect(10).to(beGreaterThan(8))\n": "expect(10) > 8\n",
+            "↓expect(10).to(beGreaterThanOrEqualTo(10))\n": "expect(10) >= 10\n",
+            "↓expect(10).to(beLessThan(11))\n": "expect(10) < 11\n",
+            "↓expect(10).to(beLessThanOrEqualTo(10))\n": "expect(10) <= 10\n",
+            "↓expect(x).to(beIdenticalTo(x))\n": "expect(x) === x\n"
         ]
     )
-
```

Comment:
Actually, I wanted to do that but testSwiftLintAutoCorrects was failing because of that. I'm not sure what exactly is happening here.

<img width="1138" alt="screen shot 2017-01-03 at 19 15 07" src="https://cloud.githubusercontent.com/assets/6388510/21618032/3e712190-d1e9-11e6-9fa7-62cb082f0acc.png">